### DISCUSSION TITLE
【入驻】孩子们，我也来了

### DISCUSSION THREAD

**Discussion by Tachankas** [share121/inter-knot#848] [其他]
隔壁科比都来了，我怎么能不跟着呢？
![meisi](https://github.com/user-attachments/assets/a46520fa-f5e5-4e54-82bd-abb01d37d936)

**Comment by lbghhj** [comment]
哟西，真的是你啊

**Comment by aaa00073** [comment]
哟——西！！！！！！！，哟西哟西哟西哟西哟西哟西哟西哟西哟西哟西哟西哟西～

**Comment by YTYX-Github** [comment]
哇 呦西 贞 德 石 泥 呀~哈哈~哎呦

**Comment by KUNKUN4** [comment]
哇，真的是你啊

**Comment by warning-Kirov-reporting** [comment]
哇，贞德石尼鸭！

**Comment by light814** [comment]
Σ(°Д°;

**Comment by UnrealGG** [comment]
孩子们，这不好笑

**Comment by Qiling-XwX** [comment]
![DE5CEDF4CFCB53E13E82EC5C55253BE5](https://github.com/user-attachments/assets/13732253-1cc3-49a0-b043-256b98877098)
